src/getting-started/assets/index.html

```diff
@@ -214,7 +214,93 @@ <h3 class="font-weight-light">Next Steps</h3>
 
                   <!-- Debugging -->
                   <div class="tab-pane fade" id="debugging-panel" role="tabpanel" aria-labelledby="debugging-tab">
-                    <p>How to use the debugger</p>
+                    <p>
+                      One of the key features of Visual Studio Code is its great debugging support. VS Code's built-in debugger helps accelerate your edit, compile and debug loop.
+                    </p>
+                    <h3 class="font-weight-light">Start Debugging</h3>
+                    <p>
+                      There are multiple ways to start debugging a Java program. By pressing <a href="command:workbench.action.debug.start"><kbd>F5</kbd></a>, the current program will be launched. You can also start by clicking the <strong>"Debug"</strong> codelens on top of the <code>main</code> function. This is super handy when you have multiple main entries. You can also launch a program <strong>without</strong> debugging by pressing <a href="command:workbench.action.debug.run"><kbd data-os="win">Ctrl + F5</kbd><kbd data-os="mac">⌃ F5</kbd></a>.
```

Comment:
Just interesting what is the result when the following html gets rendered on Linux

```html
<kbd data-os="win">Ctrl + F5</kbd><kbd data-os="mac">⌃ F5</kbd>
```

Reply:
Same as windows:
```ts
let os = "win";
if (navigator.platform.toLowerCase().indexOf("mac") === 0) {
  os = "mac";
}

const osToHide = os === "win" ? "mac" : "win";
$(`[data-os=${osToHide}]`).hide();
```